Source/WebCore/accessibility/cocoa/AXCoreObjectCocoa.mm

```diff
@@ -67,12 +67,12 @@
     auto speakAs = this->speakAs();
 
     StringBuilder builder;
-    builder.append((speakAs & SpeakAs::SpellOut) ? "spell-out"_s : "normal"_s);
-    if (speakAs & SpeakAs::Digits)
+    builder.append(speakAs.contains(Style::SpeakAsValue::SpellOut) ? "spell-out"_s : "normal"_s);
+    if (speakAs.contains(Style::SpeakAsValue::Digits))
         builder.append(" digits"_s);
-    if (speakAs & SpeakAs::LiteralPunctuation)
+    if (speakAs.contains(Style::SpeakAsValue::LiteralPunctuation))
         builder.append(" literal-punctuation"_s);
-    if (speakAs & SpeakAs::NoPunctuation)
+    if (speakAs.contains(Style::SpeakAsValue::NoPunctuation))
         builder.append(" no-punctuation"_s);
 
     return builder.toString();
```

Comment:
Seems like this could be written more efficiently to use `makeString` to concatenate four `ASCIILiteral` values, instead of using `StringBuilder`.